packages/sync/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@atproto/sync",
+  "version": "0.1.0",
```

Comment:
We can probably set this to `0.0.0` while adding a changeset for a minor version bump.

Reply:
yeah i'm game